vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import { URL, fileURLToPath } from 'url';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    include: ['**/*.vitest.ts'],
+    coverage: {
+      reporter: ['lcov', 'text'],
+    },
+  },
+  resolve: {
+    alias: [{ find: '@/', replacement: fileURLToPath(new URL('./', import.meta.url)) }],
```

Comment:
Do we need this ? also currently it is a shorthand for the project root, not `src` which is usually the goal.